### ISSUE TITLE
IndexError at / single positional indexer is out-of-bounds

### ISSUE THREAD

**Issue by yash2231** [BrianRuizy/covid-19-dashboard#6]
```

Request Method: | GET
-- | --
http://127.0.0.1:8000/
3.0.3
IndexError
single positional indexer is out-of-bounds
C:\Users\admin\AppData\Local\Programs\Python\Python38\lib\site-packages\pandas\core\indexing.py in _validate_integer, line 2063
C:\Users\admin\AppData\Local\Programs\Python\Python38\python.exe
3.8.2
['E:\\Django\\covid-19-dashboard-master',  'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python38\\python38.zip',  'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python38\\DLLs',  'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python38\\lib',  'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python38',  'C:\\Users\\admin\\AppData\\Roaming\\Python\\Python38\\site-packages',  'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages',  'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\python_telegram_bot-12.5-py3.8.egg',  'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\decorator-4.4.2-py3.8.egg',  'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\win32',  'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\win32\\lib',  'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\Pythonwin']
Sun, 19 Apr 2020 15:50:46 +0000
```

**Help me out.**

**Comment by BrianRuizy** [comment]
Can you elaborate on what you're trying to do, and how the error was encountered?
Thanks

**Comment by BrianRuizy** [comment]
Closing, no context

**Comment by BrianRuizy** [comment]
Issue not fixed -> #22
